packages/grafana-toolkit/src/cli/utils/execLine.ts

```diff
@@ -0,0 +1,14 @@
+import { exec } from 'child_process';
+
+const execLine = async (command: string): Promise<string> => {
```

Comment:
Maybe use https://github.com/sindresorhus/execa instead of adding this util? We already use exaca in toolkit

Reply:
Yes :). I can.